docs/_static/didman/v1.yaml

```diff
@@ -302,3 +336,12 @@ components:
         website:
           type: string
           description: URL of the public website of this Service Provider. Can point to a Nuts specific page with more information about the node and how to contact.
+    OrganizationSearchResult:
+      type: object
+      description: An entry resulting from the organozation search.
+      properties:
+        organization:
+          type: object
+          description: An object describing the found entity, modelled as a concept as specified by VCR's OpenAPI specification.
```

Comment:
Can you add a reference here as well, or at least an example?

Reply:
Added a link to https://nuts-node.readthedocs.io/en/latest/pages/development/3-vc.html